create_infotexts.py

```diff
@@ -374,15 +375,44 @@ def process_depicted_place(self, place_string, places_mapping):
                 place_as_wikitext = place_string
 
             # Don't forget to add the commons categories, even though only wikidata is used in depicted people field
-                if places_mapping[place_string].get('commonscat'):
-                    self.content_cats.append(places_mapping[place_string]["commonscat"])
+            if places_mapping[place_string].get('commonscat') and \
+            places_mapping[place_string].get('commonscat') != "-" and \
+            places_mapping[place_string].get('commonscat') not in self.content_cats:
+
+                self.content_cats.append(places_mapping[place_string]["commonscat"])
 
         else:
             self.meta_cats.append("Media_contributed_by_SMVK_without_mapped_place_value")
             place_as_wikitext = place_string
 
 
+        for place in places_mapping:
+            if place.lower() in desc_string.lower():
+
+                if places_mapping[place]["wikidata"] and places_mapping[place]["wikidata"] != "-":
+                    if place_as_wikitext: # There is already a depicted place found in field <Ort, foto>
+                        place_as_wikitext += "/{{{{city|1={wikidata}}}}}".format(
+                            wikidata=places_mapping[place]["wikidata"]
+                            )
+                    else:
+                        place_as_wikitext = "{{{{city|1={wikidata}}}}}".format(
+                            wikidata=places_mapping[place]["wikidata"]
+                            )
+
+                else:
+                    if place_as_wikitext:
+                        place_as_wikitext += "/{}".format(place)
+                    else:
+                        place_as_wikitext = place
+
+                # Don't forget to add the commons categories, even though only wikidata is used in depicted people field
+                if places_mapping[place].get('commonscat') and \
+                    places_mapping[place].get('commonscat') != "-" and \
+                    places_mapping[place].get('commonscat') not in self.content_cats:
+                    self.content_cats.append(places_mapping[place]["commonscat"])
+
         self.data["depicted_place"] = place_as_wikitext
+        return place_as_wikitext
```

Comment:
No need to `return` (you are storing it in `self.data["depicted_place"]` instead)